mungegithub/mungers/approvers/owners.go

```diff
@@ -539,13 +540,14 @@ func GenerateTemplateOrFail(templ, name string, data interface{}) *string {
 // 	- how an approver can cancel their approval
 func GetMessage(ap Approvers, org, project string) *string {
 	message := GenerateTemplateOrFail(`This pull-request has been approved by: {{range $index, $approval := .ap.ListApprovals}}{{if $index}}, {{end}}{{$approval}}{{end}}
-{{- if not .ap.IsApproved}}
+{{- if (and (not .ap.IsApproved) (ne 0 (len .ap.GetCCs)) )}}
```

Comment:
should you just change `IsApproved` to `AreFilesApproved`? is there another bug that you're trying to fix here?

If we have one non-approved files, then we should have at least one cc, right?